app/Filament/Resources/AchievementSetClaimResource.php

```diff
@@ -0,0 +1,200 @@
+
+    protected static ?string $navigationLabel = 'Claims';
+
+    protected static ?int $navigationSort = 70;
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+
+            ]);
+    }
+
+    public static function table(Table $table): Table
+    {
+        return $table
+            ->columns([
+                Tables\Columns\ImageColumn::make('game.badge_url')
+                    ->label('')
+                    ->size(config('media.icon.sm.width')),
+
+                Tables\Columns\TextColumn::make('game.title')
+                    ->description(fn ($record) => $record->game->system->name)
+                    ->url(function (AchievementSetClaim $record) {
+                        if (request()->user()->can('manage', Game::class)) {
+                            return GameResource::getUrl('view', ['record' => $record->game]);
+                        }
+
+                        return route('game.show', ['game' => $record->game]);
+                    })
+                    ->searchable()
+                    ->sortable(),
+
+                Tables\Columns\TextColumn::make('user.display_name')
+                    ->url(function (AchievementSetClaim $record) {
+                        if (request()->user()->can('manage', User::class)) {
+                            return UserResource::getUrl('view', ['record' => $record->user]);
+                        }
+
+                        return route('user.show', $record->game->user);
+                    })
+                    ->searchable()
+                    ->sortable(query: function (Builder $query, string $direction): Builder {
+                        return $query
+                            ->join('UserAccounts', 'SetClaim.user_id', '=', 'UserAccounts.id')
+                            ->orderByRaw('COALESCE(UserAccounts.display_name, "") ' . $direction) // Sort by display_name, treating null as empty string.
+                            ->orderBy('UserAccounts.User', $direction);
+                    }),
+
+                Tables\Columns\TextColumn::make('ClaimType')
+                    ->label('Claim Type')
+                    ->formatStateUsing(fn ($record) => ClaimType::toString($record->ClaimType)),
+
+                Tables\Columns\TextColumn::make('Status')
+                    ->formatStateUsing(fn ($record) => ClaimStatus::toString($record->Status)),
+
+                Tables\Columns\TextColumn::make('SetType')
+                    ->label('Set Type')
+                    ->formatStateUsing(fn ($record) => ClaimSetType::toString($record->SetType)),
+
+                Tables\Columns\TextColumn::make('Special')
+                    ->formatStateUsing(fn ($record) => ClaimSpecial::toString($record->Special)),
+
+                Tables\Columns\TextColumn::make('Created')
+                    ->label('Claimed At')
+                    ->dateTime('d M Y')
+                    ->sortable(),
```

Comment:
The default sort on the old page was "Newest Claim first". I don't think the new page has any default sort.

Reply:
Updated in latest.